src/cmd/package.go

```diff
@@ -104,9 +104,9 @@ var packageInspectCmd = &cobra.Command{
 	Aliases: []string{"i"},
 	Short:   lang.CmdPackageInspectShort,
 	Long:    lang.CmdPackageInspectLong,
-	Args:    cobra.MaximumNArgs(1),
+	Args:    cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:    cobra.MaximumNArgs(1),
```

`zarf package inspect` is a thing (with no args)